tests/TestFramework/Config/InfectionConfigTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+namespace Infection\Tests\TestFramework\Config;
+
+use Infection\Config\InfectionConfig;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+
+class InfectionConfigTest extends TestCase
+{
+    /** @var InfectionConfig */
+    private $testSubject = null;
+
+    /** @var Filesystem */
+    private $mockFilesystem = null;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->mockFilesystem = $this->getMockBuilder(Filesystem::class)
```

Comment:
Should we use `Mockery` everywhere to be consistent?
Do we need a mock here?

Reply:
The testSubject (InfectionConfig) requires a Filesystem object as a construct parameter. Haven't used Mockery before, but I presume it can be converted over easily enough. Personally, this is such a simple mock, I see no reason why to not just use PHPUnit's existing system.

Reply:
Move to the other file. Since the other file was already using a Filesystem object, I just updated the testing to use it.